spec/features/manage_batches_spec.rb

```diff
@@ -0,0 +1,108 @@
+require "rails_helper"
+
+require "rake"
+
+RSpec.feature "Manage batches", type: :feature do
+  before do
+    Rails.application.load_tasks
+    Timecop.freeze(Time.zone.local(2024, 2, 29)) # so we don't worry about expiry dates
+  end
+
+  after { Timecop.return }
+
+  scenario "Add a new batch" do
+    given_my_team_is_running_an_hpv_vaccination_campaign
+
+    when_i_manage_vaccines
+    then_i_see_an_hpv_vaccine_with_no_batches_set_up
+
+    when_i_add_a_new_batch
+    then_i_see_the_batch_i_just_added_on_the_vaccines_page
+  end
+
+  def given_my_team_is_running_an_hpv_vaccination_campaign
+    team = build(:team) # get some fake data
+
+    suppress_output do
+      Rake::Task["add_new_hpv_team"].invoke(
+        team.email,
+        team.name,
+        team.phone,
+        team.ods_code,
+        team.privacy_policy_url,
+        team.reply_to_id
+      )
+    end
+
+    created_team = Team.find_by(email: team.email)
+
+    suppress_output do
+      Rake::Task["add_new_user"].invoke(
+        "nurse.testy@example.com",
+        "nurse.testy@example.com",
+        "Nurse Testy",
+        created_team.id,
+        ""
+      )
+    end
```

Comment:
This is not the ideal way to set up stuff. I have some testing improvements in a branch that I'd like to adopt across. I'll update this after merging. 👍